### PR TITLE
Add support for Prisma Bytes and GraphQL scalar Byte

### PR DESCRIPTION
**Pull request by Tobbe**
It's now possible to use the Prisma `Bytes` data type [prisma-schema-reference](https://www.prisma.io/docs/orm/reference/prisma-schema-reference#bytes). Typically used for object storage (think small thumbnail images or avatars)

In adding support for Prisma Bytes, I also added the GraphQL scalar Byte

Fixes https://github.com/redwoodjs/redwood/issues/9773
Fixes https://github.com/redwoodjs/redwood/issues/5841

I've confirmed both the issue and the fix with both Postgres and SQLite

### CODE REVIEW

File: __fixtures__/example-todo-main/web/src/graphql/graphql.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export type Scalars = {
   Int: { input: number; output: number; }
   Float: { input: number; output: number; }
   BigInt: { input: any; output: any; }
+  Byte: { input: any; output: any; }
```
Comment (dthyresson) [inline __fixtures__/example-todo-main/web/src/graphql/graphql.ts:18]: Is it Byte or Bytes?

```
   const prismaScalarToTsType = {
     String: 'string',
 @@ -308,6 +308,7 @@ export const mapPrismaScalarToPagePropTsType = (scalarType) => {
     Float: 'number',
     Decimal: 'number',
     DateTime: 'string',
     Bytes: 'Buffer',
```

Comment (dthyresson) [reply __fixtures__/example-todo-main/web/src/graphql/graphql.ts:18]: Rather, I should ask: why is it Byte singular and not Bytes?

Comment (Tobbe) [reply __fixtures__/example-todo-main/web/src/graphql/graphql.ts:18]: The GraphQL type is `Byte`
The Prisma type is `Bytes`
The TS type is `Buffer`

Comment (dthyresson) [reply __fixtures__/example-todo-main/web/src/graphql/graphql.ts:18]: Oh, in Prisma is Bytes and GraphQL scalar is Byte?

Comment (dthyresson) [reply __fixtures__/example-todo-main/web/src/graphql/graphql.ts:18]: I see now: ```
   const prismaTypeToGraphqlType = {
     Json: 'JSON',
     Decimal: 'Float',
     Bytes: 'Byte',```

File: packages/cli/src/commands/generate/sdl/__tests__/sdl.test.js [hunk 2]
```diff
@@ -2,6 +2,24 @@ globalThis.__dirname = __dirname
 
 globalThis.mockFs = false
 
+// jest.mock('console', () => {
```
Comment (Tobbe) [inline packages/cli/src/commands/generate/sdl/__tests__/sdl.test.js:5]: Need to clean this up

**Review by dthyresson — APPROVED** [review]
Awesome work. Clever way to get the Bytes / Byte / Buffer values in tests and scenarios. That was one big stumbling block when I tried to implement year + ago.

LGTM!

**Pull request merged**